<?xml version="1.0"?>
<launch>

    <arg name="print" default="true"/>
    
    <node pkg="mybot_pkg" type="odom_node" name="odom_node" output="screen">
        <!-- frames -->
        <param name="odom_frame" type="string" value="odom" />
        <param name="base_frame" type="string" value="base_link" />

        <!-- topics -->
        <param name="pose_topic" type="string" value="/pose" />
        <param name="twist_topic" type="string" value="/twist" />
        <param name="odom_topic" type="string" value="/odom" />

        <!-- publish rate -->
        <param name="publish_rate" type="double" value="20.0" />

        <!-- print on terminal-->
        <param name="print_on_terminal" type="bool" value="$(arg print)" />

    </node>

</launch>